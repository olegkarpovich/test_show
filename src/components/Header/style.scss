#header-block {
  width: 100%;
  height: 40px;

  #home-button {
    position: absolute;
    left: 40px;
    width: 80px;
    background: #337AB7;
    color: #fff;
    vertical-align: middle;
    padding: 10px 0;
    text-align: center;
    text-decoration: none;
  }

  #home-button:hover {
    background: #00FFFF;
    cursor: pointer;
  }
}