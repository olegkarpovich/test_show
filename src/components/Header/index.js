import React from "react";
import { NavLink } from "react-router-dom";
import './style.scss';

function Header() {
    return (
        <nav id="header-block">
            <NavLink id="home-button" exact activeClassName="active" to="/">
                Home
            </NavLink>
        </nav>
    );
}
export default Header;
