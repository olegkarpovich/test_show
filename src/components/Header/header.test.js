import React from 'react';
import { render, act } from '@testing-library/react';
import { BrowserRouter } from "react-router-dom";
import Header from './index';

const component = () => (
    <BrowserRouter>
        <Header />
    </BrowserRouter>
);

describe('Header component', () => {
    it('should render header', () => {
        const { container } = render(component());

        expect(container).toMatchSnapshot();
    });
});