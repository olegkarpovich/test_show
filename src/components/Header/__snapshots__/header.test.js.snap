// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpisodeItem component should render episode details page 1`] = `
<div>
  <nav
    id="header-block"
  >
    <a
      aria-current="page"
      class="active"
      href="/"
      id="home-button"
    >
      Home
    </a>
  </nav>
</div>
`;

exports[`Header component should render header 1`] = `
<div>
  <nav
    id="header-block"
  >
    <a
      aria-current="page"
      class="active"
      href="/"
      id="home-button"
    >
      Home
    </a>
  </nav>
</div>
`;
